utils/aws/s3.ts

```diff
@@ -0,0 +1,41 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const S3 = new aws.S3();
+
+/* 
+
+Uploads a file to the AWS S3 bucket corresponding to AWS_BUCKET_NAME in .env.
+    * filePath: the path of the file to upload
+    * returns: upload response if successful, error object otherwise
+
+Sample usage:
+    myFile = "~/documents/my_file.txt"
+    res = uploadToS3(myFile)
+
+*/
+
+const uploadToS3 = (filePath: string): void => {
+    aws.config.update({
```

Comment:
I don't think this code needs to be in this function here. It can be set outside it. I think it can be removed after Victor updates #12 and merges it in

Reply:
Is this what you had in mind? https://github.com/uwblueprint/social-diversity-for-children/pull/11/commits/4156ac1af5483819aca96900b2314dc72f6a486e